### DISCUSSION TITLE
Resources in Podman

### DISCUSSION THREAD

**Discussion by ashley-cui** [Homebrew/discussions#2132] [Writing Formulae/Casks]
We recently added gvproxy in a resource block for podman. I was wondering if we could also add an upstream branch of qemu in a resource block on m1. This is because this branch has patches that are not in master, that we need for podman on m1 macs. We would remove this resource block and add vanilla qemu as a dependency as soon as those patches are merged.
Let me know if this would work !

cc @baude

**Comment by SMillerDev** [comment]
If they're in merge requests upstream it would be preferable to patch the qemu formula.

**Reply by ashley-cui** [reply]
The patches are submitted but don't apply cleanly as of now. That's why we're looking at resource blocks

**Reply by carlocab** [reply]
You can backport patches and embed them in the formula or host them at Homebrew/formula-patches. The latter is preferred for all but the most trivial patches (e.g. a missing `#include`).

**Reply by ashley-cui** [reply]
Okay thank you! I'll take a look and see what I can do
